config/fieldparams/mainnet.go

```diff
@@ -34,6 +34,7 @@ const (
 	BlobSidecarSize                       = 131928        // defined to match blob sidecar size in bazel ssz codegen
 	KzgCommitmentInclusionProofDepth      = 17            // Merkle proof depth for blob_kzg_commitments list item
 	NextSyncCommitteeBranchDepth          = 5             // NextSyncCommitteeBranchDepth defines the depth of the next sync committee branch.
+	NextSyncCommitteeBranchDepthElectra   = 6             // NextSyncCommitteeBranchDepthElectra defines the depth of the next sync committee branch.
```

Comment:
removed the "for Electra" from the comment as other Electra constants don't have it either